src/main/java/com/vaadin/recipes/recipe/alwayseditablelabel/AlwaysEditableLabel.java

```diff
@@ -9,18 +9,17 @@
 
 @Route("always-editable-label")
 @Metadata(
-        howdoI = "Create an always-editable Label",
-        description = "Create a Label component that is always in editable mode.",
+        howdoI = "Create an always-editable Span",
```

Comment:
I don't think "Span" is the important part here. Should it say "label"?

Reply:
Done